constants/index.tsx

```diff
@@ -0,0 +1,5 @@
+export const COOKIES_EXPIRATION_TIME = 60 * 60 * 24 * 30; // 30 days
+export const COOKIE_CHECK_ID_NAME = 'onfido-check-id';
+export const LOCALSTORAGE_USER_DATA_NAME = 'onfido-user-data';
+export const SHORT_POLLING_INTERVAL = 1000;
+export const LONG_POLLING_INTERVAL = 30000;
```

Comment:
Presumably 30_000 here means 30 seconds, but I'd clarify these lines with comments. And then maybe describe the difference between when SHORT_ or LONG_ would be used.

Reply:
Thanks done!